.teamcity/common/performance-test-extensions.kt

```diff
@@ -75,22 +75,27 @@ fun BuildSteps.substDirOnWindows(os: Os, buildCache: BuildCache) {
             executionMode = BuildStep.ExecutionMode.ALWAYS
             scriptContent = """subst p: "%teamcity.build.checkoutDir%" """
         }
-        // Gradle detects overlapping outputs when running first on a subst drive and then in the original location.
-        // Even when running clean builds on CI, we don't run clean in buildSrc, so there may be stale leftover files there.
-        // This means that we need to clean buildSrc before running for the first time on the subst drive
-        // and before running the first time on the original location again.
-        gradleWrapper {
-            name = "CLEAN_BUILD_SRC_ON_SUBST_DRIVE"
-            tasks = "clean"
-            workingDir = "P:/build-logic"
-            executionMode = BuildStep.ExecutionMode.ALWAYS
-            gradleWrapperPath = "../"
-            gradleParams = (
-                buildToolGradleParameters() +
-                    buildScanTag("PerformanceTest") +
-                    buildCache.gradleParameters(os)
-                ).joinToString(separator = " ")
-        }
+        cleanBuildLogicBuild("build-logic-commons", buildCache, os)
```

Comment:
This also needs to happen in `removeSubstDirOnWindows`.

Reply:
Thanks. Will fix.